games/c.yaml

```diff
@@ -385,7 +385,7 @@
     - http://raw.github.com/cxong/cdogs-sdl/master/wiki/images/thumbs/3.png
     - http://raw.github.com/cxong/cdogs-sdl/master/wiki/images/thumbs/4.png
     video:
-      youtube: fSIOdqZFM48
+      youtube: DysXfUyw58M
```

Comment:
The other one is in German and 2.5 years old..

Just noticed that you released a new trailer though, I'll put that one up.